spec-tests/runners/kzg.rs

```diff
@@ -0,0 +1,183 @@
+use crate::{
+    test_case::TestCase,
+    test_utils::{load_yaml, Error},
+};
+use ethereum_consensus::deneb::{
+    mainnet::Blob,
+    polynomial_commitments::{
+        blob_to_kzg_commitment, compute_kzg_proof, kzg_settings_from_json, verify_kzg_proof,
+        Error as PolynomialCommitmentsError, FieldElement, KzgCommitment, KzgProof, KzgSettings,
+        ProofAndEvaluation,
+    },
+    presets::TRUSTED_SETUP_JSON,
+};
+
+pub fn dispatch(test: &TestCase) -> Result<(), Error> {
+    let kzg_settings = kzg_settings_from_json(TRUSTED_SETUP_JSON)?;
```

Comment:
just a heads up; after #374, we can get the KZG settings from the context, which is already reachable from `TestCase`

```suggestion
    let kzg_settings = &test.context().kzg_settings;
```

and then we can drop some imports from this file